### PR TITLE
add SomeService and SomeServiceTest

### PR DESCRIPTION
**Pull request by octocat-jedi**

### CODE REVIEW

File: app/src/main/java/com/ex/service/SomeService.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.ex.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class SomeService {
```
Comment (octocat-jedi) [inline app/src/main/java/com/ex/service/SomeService.java:6]: MINOR Move this left curly brace to the beginning of next line of code. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)

File: app/src/main/java/com/ex/service/SomeService.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	
+	public int add(int x, int y){
```
Comment (octocat-jedi) [inline app/src/main/java/com/ex/service/SomeService.java:8]: MINOR Move this left curly brace to the beginning of next line of code. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
MINOR Make this line start at column 3. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AIndentationCheck)

File: app/src/main/java/com/ex/service/SomeService.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+		return x + y;
```
Comment (octocat-jedi) [inline app/src/main/java/com/ex/service/SomeService.java:9]: MINOR Make this line start at column 5. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AIndentationCheck)

**Comment by octocat-jedi** [comment]
SonarQube analysis reported 6 issues
- MINOR 6 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MINOR [service](https://github.com/revature-training/boot-app/blob/1878f47724a5a64c47529f408da350d5024f31f3/app/src/main/java/com/ex/service): Add a 'package-info.java' file to document the 'service' package [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AS1228)
2. MINOR [SomeService.java](https://github.com/revature-training/boot-app/blob/1878f47724a5a64c47529f408da350d5024f31f3/app/src/main/java/com/ex/service/SomeService.java): Replace all tab characters in this file by sequences of white-spaces. [rule](http://sonar.taylorkemper.com/sonar/coding_rules#rule_key=squid%3AS00105)

**Review by tjkemper — APPROVED** [review]
lgtm

**Pull request merged**